aptos-move/aptos-release-builder/src/components/framework.rs

```diff
@@ -24,7 +25,67 @@ pub fn generate_upgrade_proposals(
     next_execution_hash: Vec<u8>,
 ) -> Result<Vec<(String, String)>> {
     const APTOS_GIT_PATH: &str = "https://github.com/aptos-labs/aptos-core.git";
+    generate_upgrade_proposals_with_repo(config, is_testnet, next_execution_hash, APTOS_GIT_PATH)
+}
+
+pub fn generate_upgrade_proposals_with_repo(
```

Comment:
Is this meant to be a public method? Or just an internal code reorg for `generate_update_proposals`? In the latter case, it's not clear what you win by splitting out everything except one const declaration.